### DISCUSSION TITLE
細菌會說話嗎？來自生物世界最早的聲音！

### DISCUSSION THREAD

**Discussion by Tuoresa** [1101-GS4719-C/Python-Community-GS4719-C#1071] [General]
「生物的聲音最早來自細菌，向周遭水域釋出幾不可聞的低語，或嘆息或愉悅。」

這篇報導主要闡述科學家嘗試「聽見」細菌的聲音，將細菌放在極微小的支柱頂端，
當它的細胞表面的任何振動，都會牽動支柱，科學家也因此可以用雷射光鎖定支柱，記錄並測量這些動態。

透過這項實驗發現細菌時時刻刻顫動著，製造出持續性的聲波。
這些聲波大約只有五奈米，比人類說話時聲帶振動的幅度小五十萬倍。

有點像在觀察細菌發抖的實驗，
莫名可愛哈哈
雖然現在研究的相關文件極少，也不確定能有什麼樣的突破與應用
但確實再次讓人意識到，這個世界仍有太多我們尚未聽見與看見的存在

把耳朵貼近桌面時，都會聽見不間斷的、此起彼落的各種小雜音，
小時候我總覺得那裏面有一個小小的世界忙碌無比(現在想起來怎麼好像偏笨

文章連結：https://pansci.asia/archives/359863

**Comment by peggy0329** [comment]
我覺得這個研究好有趣，有突破既定研究的感覺！
像是平常大家只會研究細菌的活動、變化等等，卻很少研究到細菌的「五感」
而其實文章中提及細菌的聲音是以震動製造聲波
我覺得或許可以透過這個小小的發現應用在我們未知的領域中
果然這個世界是充滿了變數呢！

**Comment by ru0372572** [comment]
我小時候也會把耳朵貼在桌子上聽！其實到現在還是不知道那些不一樣的小雜音是怎麼來的，但是好好笑，果然大家小時候想的都差不多~

**Comment by yuxinlin93** [comment]
如果細菌會講話也太酷了吧，好好奇他們會講什麼，但他們在地球那麼久了，會不會其實發展傳承下來很多很神奇的東西呢?好好奇

**Comment by Valentinaa0214** [comment]
很新奇的發現欸！若是未來人類能夠將他們的聲音加以翻譯解讀，說不定能夠為我們探尋地球的過去與未來有很大的幫助！

**Comment by orianna901001** [comment]
這個主題也太有趣了吧，原來我身上這麼多的細菌都在偷偷聊天，希望未來有人可以翻譯一下它們聊天內容XD
